rflx/pyrflx/typevalue.py

```diff
@@ -929,8 +932,8 @@ def valid_path(value_range: ValueRange) -> bool:
                 isinstance(expr_tuple.evaluated_expression, ValueRange)
                 and isinstance(expr_tuple.expression, ValueRange)
                 and (
-                    not isinstance(expr_tuple.evaluated_expression.lower, Number)
-                    or not isinstance(expr_tuple.evaluated_expression.upper, Number)
+                    not isinstance(expr_tuple.evaluated_expression.lower.simplified(), Number)
+                    or not isinstance(expr_tuple.evaluated_expression.upper.simplified(), Number)
```

Comment:
This change seems to be unnecessary.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/424/commits/074b1678c79822fef9d65d2150a02e01e0b43e0d.